openlibrary/plugins/openlibrary/api.py

```diff
@@ -487,3 +488,35 @@ def response(msg, status="success"):
             }), content_type="application/json")
 
         return response('Observations removed')
+
+
+class works(delegate.page):
+    path = r"/works/OL(\d+)W"
```

Comment:
That way we don't have to reconstruct the work_id twice
```suggestion
    path = r"/works/(OL\d+W)"
```